<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="check">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Log axis</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 cm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5 cm</string>
     </property>
    </item>
   </widget>
   <widget class="QOpenGLWidget" name="myGLWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>481</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Plot</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>10 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1 Hz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0.1 Hz</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbx">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>MM</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbx_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Z(f)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>C(f)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Z2(Z1)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ph(f)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sample</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Distance</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Function</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Measurements Plot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
